content/applications/websites/ecommerce/checkout_payment_shipping/ewallet_giftcard.rst

```diff
@@ -0,0 +1,122 @@
+   :align: center
+   :alt: eWallet configuration
+
+The :guilabel:`EWallets` smart tab in the upper-right corner shows you the number of existing
+eWallets. Click on it to access more details about each existing eWallet.
+
+Checkout
+--------
+
+On the checkout page, customers have the option to pay with their **eWallet** in the
+:guilabel:`Order Total` column, and the amount will be deducted from their **eWallet**.
+
+.. image:: ewallet_gift/ewallet-checkout.png
+   :align: center
+   :alt: eWallet checkout option
+
+Gift cards
+==========
+
+.. _ewallet_gift/gift-cards:
+
+You can generate **gift cards** which can be bought by your customers and in turn used as **payment
+methods** upon checkout. Before creating a new gift card program, it is necessary to first create
+gift cards as a **product**. To do so, go to :menuselection:`Website app --> eCommerce --> Products`
+and :guilabel:`Create` a gift card **product**. The following configuration options are available:
+
+- :guilabel:`Can be Sold`: enabled;
+- :guilabel:`Product Type`: set as :guilabel:`Service`;
+- :guilabel:`Invoicing Policy`: set as :guilabel:`Prepaid/Fixed Price`;
+- :guilabel:`Create on Order`: select :guilabel:`Nothing`;
+- :guilabel:`Sales Price`: type in the **value** of the gift card.
+
+.. note::
+   If you wish to have **gift cards** of different values, repeat the process and modify the
+   :guilabel:`Sales Price` accordingly.
+
+Once the gift card **product** is created, go to :menuselection:`Website app --> eCommerce --> Gift
+cards & eWallet` to :guilabel:`Create` a gift cards **program**. The following configuration options
+are available:
+
+- :guilabel:`Program Name`: enter the name you want;
+- :guilabel:`Program Type`: select :guilabel:`Gift Card`;
+- :guilabel:`Gift Cards Products`: select the gift card **product(s)** created earlier;
+- :guilabel:`Email template`: select the default :guilabel:`Gift Card: Gift Card Information`
+  template, or create your own by clicking the field and :guilabel:`Search More`;
+- :guilabel:`Print Report`: select :guilabel:`Gift Card`;
+- :guilabel:`Currency`: select the currency you wish to use for that gift card program;
+- :guilabel:`Company`: the company in which the program is valid and available;
+- :guilabel:`Available On`: the different stores where the program is valid and available;
+- :guilabel:`Website`: the website on which the program is valid and available, leave empty for all;
+- :guilabel:`Point of Sale`: :abbr:`PoS (Point of Sale)` in which the program is valid and
+  available, leave empty for all.
+
+.. image:: ewallet_gift/giftcard-config.png
+   :align: center
+   :alt: Gift card configuration
+
+The :guilabel:`Gift Cards` smart tab in the upper-right corner shows you the number of existing
+gift cards. Click on it to access more details about each existing gift card.
+
+Checkout
+--------
+
+On the checkout page, customers have the option to pay with their **gift card** in the
```

Comment:
```suggestion
When finalising their purchases, on the checkout page, customers have the option to pay with their **gift card** in the
```

Reply:
Good suggestion, no need for the comma though (so I'm leaving it out)

Reply:
not resolved from my side